### PR TITLE
Update API Subgroup Focus description

### PR DESCRIPTION
**Pull request by sandersms**
Signed-off-by: Mark Sanders <Mark_S_Sanders@dell.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,17 +6,16 @@
...
+OPI subgroup dealing with APIs.  The focus of the subgroup is:
 
 - Define the object models for each of the components and services on the platform.
   - Define the layering of the capabilities that are abstracted
 - Define both host system facing and control/management facing APIs
   - Discovery and Service Advertisement API
   - Capabilities/Capacity Advertisement API
     - Information such as Processor Cores available, Bandwidth, Memory, etc.
+  - Configuration API
+  - Consumption API for the data connections to the Host Compute
```
Comment (glimchb) [inline README.md:18]: line 22 storage and gw need indentation as well ?

File: README.md [hunk 2]
```diff
@@ -25,8 +24,13 @@ The focus of the subgroup is:
   - Telemetry
   - [AI/ML](aiml/README.md)
   - others
+- Define recommended industry APIs to be used.
+  - Examples (OVS DB, OpenConfig, VPP)
+- Identify gaps in APIs and recommend extensions for those capabilities
 - Create/Define the Backend/Southbound facing interface APIs
 
+The API subgroup does not want to re-invent any industry standard APIs that already exist.  Standard APIs such as OVS DB, OpenConfig, VPP, etc. that are used for configuration of service capabilities will be reviewed and examined for gaps.  Gaps in capabilities will be addressed either by defining extensions to existing APIs or new APIs will need to be defined (such as Storage APIs).
+
```
Comment (glimchb) [inline README.md:33]: should API group also produce reference implementations of some kind ? 
like for storage we have SPDK-OPI bridge app + OPI go client
same for security with IPSEC...

**Review by glimchb — COMMENTED** [review]
few minor suggestions

File: README.md (hunk 1 above)
Comment (sandersms) [reply README.md:18]: They look like they are already indented under the Taxonomy list.

File: README.md (hunk 2 above)
Comment (sandersms) [reply README.md:38]: Amended the checkin

File: README.md [hunk 3]
```diff
@@ -25,7 +24,17 @@ The focus of the subgroup is:
...
   - others
+- Define recommended industry APIs to be used.
+  - Examples (OVS DB, OpenConfig, VPP)
+- Identify gaps in APIs and recommend extensions for those capabilities
 - Create/Define the Backend/Southbound facing interface APIs
+- Produce protocol buffer (protobuf) definitions.
+- Produce reference orchestration client to support testing of the API.
+- Produce reference translation/bridge functions for the OPI API to SDK Backend.
+
+The API subgroup does not want to re-invent any industry standard APIs that already exist.  Standard APIs such as OVS DB, OpenConfig, VPP, etc. that are used for configuration of service capabilities will be reviewed and examined for gaps.  Gaps in capabilities will be addressed either by defining extensions to existing APIs or new APIs will need to be defined (such as Storage APIs).
```
Comment (glimchb) [inline README.md:35]: duplication ?

Comment (sandersms) [reply README.md:34]: Removed the duplicate.

Comment (sandersms) [reply README.md:34]: Removed the duplicate in the amended change.

**Review by glimchb — APPROVED** [review]
yep, all dup is gone, approved

File: README.md (hunk 3 above)
Comment (sandersms) [reply README.md]: amended again with the merge hickup fix.

**Pull request merged**